py-server/main.py

```diff
@@ -19,7 +21,11 @@
 
 # Create chat models. A model is just a class that implements the chat method
 # in order to respond to the chat history and any other context passed in.
-MODES = {"default": DefaultModel(), "fake": FakeModel()}
+MODES = {
+    "default": DefaultModel(),
+    "fake": FakeModel(),
+    "history tutor": HistoryTutor(),
```

Comment:
s/history tutor/history_tutor (and in the client)

Reply:
done -- I'll stick with the ugly string in the client for now & prettify it in a follow up.
I added a TODO